cluster/ci/monitoring/mixins/grafana_dashboards/plank.jsonnet

```diff
@@ -18,17 +19,26 @@ dashboard.new(
         time_from='now-1h',
         schemaVersion=18,
       )
+.addTemplate(template.new(
+        'cluster',
+        'prometheus',
+        std.format('label_values(prowjobs{job="plank"}, %s)', 'cluster'),
+        label='cluster',
+        refresh='time',
+        allValues='.*',
+        includeAll=true,
+    ))
 .addPanel(
     (graphPanel.new(
-        'number of Prow jobs by type',
-        description='sum(prowjobs{job="plank"}) by (type)',
+        'number of Prow jobs by type with cluster=${cluster}',
+        description='sum(rate(prowjobs{job="plank", cluster=~"${cluster}"}[5m])) by (type)',
```

Comment:
Why? If we want a rate, we could add a dashboard, but that's not the only useful way to look at this. The title is also wrong if you do thi

Reply:
Removed.